org-code-javabuilder/.gitignore

```diff
@@ -10,3 +10,5 @@ app/.project
 *.classpath
 .settings
 .project
+
+.vscode/*
```

Comment:
(very minor) Not sure if you care of not, but this file doesn't end in a newline.

Btw, do you have checkstyle configured?  It probably wouldn't have caught this but it'd be good to get it set up before adding lots more Java code.